packages/dx-react-scheduler-material-ui/src/templates/drag-drop.jsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+import { AppointmentContent } from './appointment/appointment-content';
+import { Appointment } from './appointment/appointment';
+
+const styles = theme => ({
+  container: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    height: '100%',
+    width: '100%',
+    cursor: 'move',
+  },
+  draft: {
+    boxShadow: '0px 1px 5px 0px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 3px 1px -2px rgba(0,0,0,0.12)',
```

Comment:
Can we use shadow from [default](https://material-ui.com/customization/default-theme/) MUI theme?